libs/sr/localization/assets/locales/en/lightCone.json

```diff
@@ -0,0 +1,41 @@
+{
+  "editor": {
+    "title": "Light Cone Editor",
+    "unknownLightCone": "Unknown Light Cone",
+    "lightConeName": "Light Cone Name",
+    "levelSelect": "Select Level",
+    "preview": "Light Cone Preview",
+    "beforeEdit": "Before Edit",
+    "btnSave": "Save Light Cone",
+    "btnAdd": "Add Light Cone",
+    "btnClear": "Clear",
+    "btnRandom": "Randomize",
```

Comment:
Clear + Random should go into the `common` namespace https://github.com/frzyc/genshin-optimizer/blob/master/libs/common/localization/assets/locales/en/common.json